gradle.properties

```diff
@@ -18,9 +18,9 @@ dockerComposePluginVersion=0.6.6
 eventuateCdcVersion=0.4.0.RELEASE
 eventuateCommonVersion=0.9.0.RC4
 eventuateMessagingKafkaVersion=0.9.0.RC4
-eventuateMessagingActiveMQVersion=0.6.0.RC4
-eventuateMessagingRabbitMQVersion=0.7.0.RC4
-eventuateMessagingRedisVersion=0.7.0.RC3
+eventuateMessagingActiveMQVersion=0.5.0-SNAPSHOT
```

Comment:
It would be helpful if PRs references BUILD-SNAPSHOTs instead.
And these projects published BUILD-SNAPSHOTs :-)